src/test/java/org/jenkinsci/plugins/docker/workflow/FromFingerprintStepTest.java

```diff
@@ -126,10 +122,6 @@ public void evaluate() throws Throwable {
                 String ancestorImageId = client.inspect(new EnvVars(), fromImage, ".Id");
                 story.j.assertLogContains("built from-with-arg", b);
                 story.j.assertLogContains(ancestorImageId.replaceFirst("^sha256:", "").substring(0, 12), b);
-                Fingerprint f = DockerFingerprints.of(ancestorImageId);
```

Comment:
FWIW unless this test is failing because of the change to `Docker.groovy`, I'd leave this file unchanged. If it is failing, then maybe better to just delete the test case entirely since I think these were the main assertions.

Reply:
Agreed, there is no need to touch this test that I can see.

Reply:
The tests are failing now because the build step is not producing the fingerprints anymore. I would go ahead and remove the test completely then.

Reply:
The test can be fixed: change the `script` to directly run this step, rather than using the `docker.build` DSL method.

Reply:
@jglick I can do that. Though that would mean changing the test to directly use the Dockerfile and stop manipulating the `FROM` part in the Dockerfile.

Reply:
From a quick glance, `buildWithFROMArgs` actually looks like it was about the DSL, and thus should have been in `DockerDSLTest` to begin with (and now may be deleted, as it is testing functionality which no longer exists). This test need only cover things which the step actually does.